### PR TITLE
Chore: add March and April UA string meeting minutes

### PR DESCRIPTION
**Pull request by miketaylr**
Ready for review.

### CODE REVIEW

**Comment by karlcow** [comment]
don't forget the index too.

File: uastring/minutes/2022-04-06-minutes.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    * Not really
+    * [jQuery Reject sites (Chrome 100 version parsing error) - Public](https://docs.google.com/spreadsheets/d/1uk2EiDrsuqBGFTjT5QGhliAuenzDgfZD9EXZifzzEz8/edit#gid=0)
+        * about 100 sites still unfixed
+        * shows an “upgrade your browser” message to Chrome and Edge 100+
+        * Mike will do the outreach for top sites only
+
+### Next steps for Compat Standard’s UA section?
+    * https://compat.spec.whatwg.org/#ua-string-section
+    * Mike is still addressing bugs (slowly). Rewrote Safari UA section.
+    * When UA reductions ship, no need to describe the change. Describing the latest UA is OK.
```
Comment (karlcow) [inline uastring/minutes/2022-04-06-minutes.md:36]:
```suggestion
* https://compat.spec.whatwg.org/#ua-string-section
* Mike is still addressing bugs (slowly). Rewrote Safari UA section.
* When UA reductions ship, no need to describe the change. Describing the latest UA is OK.
```

to note make it a pre

File: uastring/minutes/2022-04-06-minutes.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+###  UA reductions?
+    * Is Google still on track to ship reduced Chrome major.0.0.0 version numbers in Chrome 101?
+    * On track yes (see I2S).
+    * Schedule: https://www.chromium.org/updates/ua-reduction/#proposed-rollout-plan
+    * Only known breakage from navigator.oscpu reduction: Netflix on Chrome OS https://bugs.chromium.org/p/chromium/issues/detail?id=1298570
+    * So far no breakage on identifying “0” in minor version (fingers crossed)
+    * Firefox UA reductions
+        * Chris has some patches to freeze OS versions and CPU architectures in Firefox UA, but Mozilla will probably wait for Chrome to ship their reductions first.
```
Comment (karlcow) [inline uastring/minutes/2022-04-06-minutes.md:44]:
```suggestion
* Is Google still on track to ship reduced Chrome major.0.0.0 version numbers in Chrome 101?
* On track yes (see I2S).
* Schedule: https://www.chromium.org/updates/ua-reduction/#proposed-rollout-plan
* Only known breakage from navigator.oscpu reduction: Netflix on Chrome OS https://bugs.chromium.org/p/chromium/issues/detail?id=1298570
* So far no breakage on identifying “0” in minor version (fingers crossed)
* Firefox UA reductions
  * Chris has some patches to freeze OS versions and CPU architectures in Firefox UA, but Mozilla will probably wait for Chrome to ship their reductions first.
```

Same

**Review by karlcow — CHANGES_REQUESTED** [review]
2 small things to have proper md-> html.

Thanks.

File: uastring/minutes/2022-04-06-minutes.md (hunk 1 above)
Comment (miketaylr) [reply uastring/minutes/2022-04-06-minutes.md:36]: > to note make it a pre

Sorry, I don't follow. Should something be wrapped in backticks, @karlcow?

Comment (karlcow) [reply uastring/minutes/2022-04-06-minutes.md:36]: currently the `*` has too many spaces it converts it as a pre instead of a list item.

Comment (karlcow) [reply uastring/minutes/2022-04-06-minutes.md:36]: haha!

to **NOT** make it a pre

Comment (karlcow) [reply uastring/minutes/2022-04-06-minutes.md:36]: Capture d’écran 2022-04-08 à 06 25 23

Comment (miketaylr) [reply uastring/minutes/2022-04-06-minutes.md:36]: Ah! Now I see. :)

And I realize I only did the rich preview for the first doc, and not the second. Oops.

**Review by karlcow — APPROVED** [review]
Thanks

**Pull request merged**